pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -805,6 +810,10 @@ func TestKillContainerGracePeriod(t *testing.T) {
 
 // TestUpdateContainerResources tests updating a container in a Pod.
 func TestUpdateContainerResources(t *testing.T) {
+	// TODO: remove this check on this PR merges: https://github.com/kubernetes/kubernetes/pull/112599
+	if goruntime.GOOS == "windows" {
+		t.Skip("Updating Pod Container Resources is not supported on Windows.")
+	}
```

Comment:
previous test has the `defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.InPlacePodVerticalScaling, true)()`. Why this test doesn't?

Reply:
@vinaykul

Reply:
There's no need. (The entire function is new, no feature gate protected code addition). Calls to this function from production code are feature gate protected ofcourse.

Reply:
this is not production code as I understand from the name of the file. Sorry I didn't look deep. Bu I'd expect something similar - windows check right next to feature gate setting.

Reply:
Sorry I was not very clear earlier. I meant updateContainerResources is a new function I added that is being unit tested, no feature gate to toggle in that function or its called function. And getPodContainerStatuses is called only for obtaining the containerId